fsharp-backend/src/LibExecution/StdLib/LibString.fs

```diff
@@ -155,24 +158,24 @@ let fns : List<BuiltInFn> =
       sqlSpec = NotYetImplementedTODO
       previewable = Pure
       deprecated = ReplacedBy(fn "" "" 0) }
-//      { name = fn "String" "toInt" 1
-//      ; parameters = [Param.make "s" TStr]
-//      ; returnType = TResult
-//      ; description =
-//       "Returns the int value of the string, wrapped in a `Ok`, or `Error <msg>` if the string contains characters other than numeric digits"
-//      ; fn =
-//         (function
-//         | _, [DStr s] ->
-//             let utf8 = Unicode_string.to_string s in
-//             ( try DResult (ResOk (DInt (Dint.of_string_exn utf8)))
-//               with e ->
-//                 error_result
-//                   ("Expected a string with only numbers, got " ^ utf8) )
-//         | args ->
-//             incorrectArgs ())
-//      ; sqlSpec = NotYetImplementedTODO
-//      ; previewable = Pure
-//      ; deprecated = NotDeprecated }
+    { name = fn "String" "toInt" 1
```

Comment:
I added this, it's on main now. You can just replace this (same notes about int/bigint data loss as before.